src/science/mathematics/linear-algebra/vector_sum.md

```diff
@@ -0,0 +1,49 @@
+## Vector Sum
+[![ndarray-badge]][ndarray]
+
+The [ndarray] crate supports a number of ways to create arrays -- in this
+example we focus on creating arrays from `std::Vec` via [`from_vec`]. Adding two
```

Comment:
Avoid personal pronouns.  Prefer to talk about the code over the author or the audience.  "this recipe focuses on..."

Reply:
Good catch